.github/workflows/integration-tests.yaml

```
@@ -0,0 +1,84 @@
+on:
+  pull_request:
```

```suggestion
on:
  workflow_dispatch:
  schedule:
    - cron: "0 0 * * *"
```
Firebase test lab limit us. so dont need to run in every request, night run is enough

---

Updated